lib/friendly_id/migration.rb

```diff
@@ -1,6 +1,8 @@
+require 'rails'
+
 migration_class =
-  if ActiveRecord::VERSION::MAJOR >= 5
-    ActiveRecord::Migration[4.2]
+  if Rails.version.start_with? "5"
```

Comment:
Won’t this break again with Rails 6?

Reply:
@parndt yea, nice pick. We can make it to check for starts with 5 OR > 5.

But Travis was failing that "Rails is not available" in next line Rails::Version.. 
Can you guide me on requiring rails at this point?

Reply:
That would be fine too, http://api.rubyonrails.org/classes/ActiveRecord/VERSION.html